pkg/common/info.go

```diff
@@ -80,18 +81,28 @@ const uvmSecurityCtxDirDefault = "/opt/confidential-containers/share/kata-contai
 // This code detects which method is being used and calls the appropriate
 // function to get the UVM information.
 
-// The environment variable scheme will go away by "General Availability"
+// The environment variable scheme will go away by "General Availability" -- SHOULD WE REMOVE THIS CODE NOW?
 // but we handle both to decouple this code and the hcsshim/gcs code.
 
 // Matching PR https://github.com/microsoft/hcsshim/pull/1708
 
 func GetUvmSecurityCtxDir() (string, error) {
-	securityContextDir := os.Getenv("UVM_SECURITY_CONTEXT_DIR")
+	securityContextDir := "" // os.Getenv("UVM_SECURITY_CONTEXT_DIR")

```

Comment:
if this env var isn't used anymore, I would update the tests and docs that mention it